File: JavaScript_12.18/1235 - De Dentro para Fora.js
/**
 * @author Josélio de S. C. Júnior <joseliojrx25@gmail.com>
 * @copyright Josélio de S. C. Júnior 2021
*/

const p = require('fs').readFileSync('/dev/stdin', 'utf8'),
l = p.split('\n'),
t = l[0];

for (let i = 1; i <= t; i++) {
    const f = l[i],
    fn = (x, y) => f.split('')
        .slice(x, y)
        .reverse()
        .join('');
    console.log(`${fn(0, f.length / 2)}${fn(f.length / 2, f.length)}`);
}

File: Lua_5.4/1235 - De Dentro para Fora.lua
--- author: Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
--- copyright: Josélio Júnior (Lunatic Fox) 2021

local t, i = tonumber(io.read()), 0
while i < t do
  local f = io.read()
  local function fn(str, x, y, sts)
    return sts and (str:sub(x, y)):reverse() or (str:reverse()):sub(x, y)
  end
  print(fn(f, 1, #f / 2, true) .. fn(f, 1, #f / 2))
  i = i + 1
end